stat/distmv/statdist.go

```diff
@@ -207,6 +207,76 @@ func (KullbackLeibler) DistUniform(l, r *Uniform) float64 {
 	return logPx - logQx
 }
 
+// Renyi is a type for computing the Rényi divergence of order α from l to r.
+// The dimensions of the input distributions must match or the function will panic.
+//
+// The Rényi divergence with α > 0, α ≠ 1 is defined as
+//  D_α(l || r) = 1/(α-1) log(\int_x l(x)^α r(x)^(1-α)dx)
```

Comment:
What is the interval of integration? Using the integration variable there is not good, so writing just `\int` is better.  The same for Bhattacharyya.